File: 1112-find-words-that-can-be-formed-by-characters/find-words-that-can-be-formed-by-characters.py
class Solution:
    def countCharacters(self, words: List[str], chars: str) -> int:
        count = 0
        for word in words:
            flag =  True
            for ch in word:
                if word.count(ch) > chars.count(ch):
                    flag = False
                    break
            if flag:
                count += len(word)
            
        return count

File: 1112-find-words-that-can-be-formed-by-characters/find-words-that-can-be-formed-by-characters.cpp
class Solution {
public:
    int countCharacters(vector<string>& words, string chars) {
        int total = 0;
        for(string word:words){
            bool flag = true;
            for(char ch : word){
                if(count(word.begin(),word.end(),ch) > count(chars.begin(),chars.end(),ch)){
                    flag = false;
                    break;
                }
            }
            if(flag){
                total += word.size();
            }
        }
        return total;
    }
};